packages/hash/api/src/model/knowledge/entity.model.ts

```diff
@@ -44,47 +38,48 @@ export type EntityModelCreateParams = {
  * @class {@link EntityModel}
  */
 export default class {
-  ownedById: string;
+  private entity: Entity;
 
-  entityId: string;
-  version: string;
   entityTypeModel: EntityTypeModel;
-  properties: object;
 
-  createdById: string;
-  updatedById: string;
-  removedById: string | undefined;
-
-  constructor({
-    ownedById,
-    entityId,
-    version,
-    entityTypeModel,
-    properties,
-    createdById,
-    updatedById,
-    removedById,
-  }: EntityModelConstructorParams) {
-    this.ownedById = ownedById;
-
-    this.entityId = entityId;
-    this.version = version;
-    this.entityTypeModel = entityTypeModel;
-    this.properties = properties;
+  get baseId(): string {
+    return this.metadata.editionId.baseId;
+  }
+
+  get entityUuid(): string {
+    const [_, entityUuid] = this.baseId.split("%") as [string, string];
+
+    return entityUuid;
+  }
+
+  get ownedById(): string {
+    const [ownedById, _] = this.baseId.split("%") as [string, string];
+
+    return ownedById;
+  }
```

Comment:
As part of the sub-graph changes, we no longer have a single concept of an `entityId` on the Graph API layer. Some endpoints expect the `baseId` whilst others want the `entityUuid`, so I've made these accessible on the top-level of the model classes here.

The removal of `entity.entityId` causes linting errors which are beyond the scope of this PR to address, as they need to be replaced by a case-by-case basis. I've created [a follow-up task](https://app.asana.com/0/0/1203361740074435/f) for this. @hashintel/backend let me know if you have any thoughts/concerns regarding this.

Reply:
It's nice that these are getters on the class, and the steps here make sense to make the incremental changes we want :+1: